application-analytics-ui/pom.xml

```diff
@@ -35,25 +35,19 @@
     productivity by using real-time statistics.
   </description>
   <properties>
-    <!-- Name to display by the Extension Manager -->
-    <!-- The list of documents that are not covered by the extension license. The user that installs the extension has
-    full access to these documents (view, edit, delete) without having a license, so use this property wisely. -->
-    <xwiki.extension.licensing.excludedDocuments>
-      Analytics.Code.Configuration
-    </xwiki.extension.licensing.excludedDocuments>
     <xwiki.extension.name>Analytics Application (Pro)</xwiki.extension.name>
     <xwiki.extension.category>application</xwiki.extension.category>
 
     <!-- The list of documents that have an implicit unlimited free license. The users can view these documents without
      buying a license or getting a trial license, but they cannot edit or delete them. -->
     <xwiki.extension.licensing.publicDocuments>
-      Analytics.WebHome
     </xwiki.extension.licensing.publicDocuments>
     <!-- Name to display by the Extension Manager -->
```

Comment:
I know it's related to this PR, but this comment is misplaced. It should be before `<xwiki.extension.name>`